doc/input-output-reference/src/standard-output-reports/output-table-summaryreports.tex

```diff
@@ -425,6 +425,10 @@ \subsubsection{Predefined Annual Summary Reports}\label{predefined-annual-summar
 
 The Life Cycle Cost Report provides the a summary of the information from the Life Cycle Cost calculations.  This report shows the costs and the timing of costs, often called “cash flows,” along with the present value in several different tables. The tabular results show the present value of all current and future costs. The key used to obtain this report is LifeCycleCostReport.
 
+\paragraph{Heat Emissions Report}\label{HeatEmissionsReport}
+
+The Heat Emissions Report provides the a summary of the building heat emission to the ambient calculations. This report shows the heat emissions from buildings by surface heat transfer, zone exhaust, zone relief, HVAC relief, HVAC heat rejection, as well as in total. The key used to obtain this report is HeatEmissionsSummary.
```

Comment:
provides ~the~ a summary (and please fix this in the life cycle cost report above, thanks)
. . . to the ambient ~calculations~ *air*.
. . . heat rejection~, as well as in total~.